src/access/acp.ts

```diff
@@ -407,3 +410,15 @@ export function internal_getActorAccessAll(
   });
   return result;
 }
+
+export function internal_getGroupAccessAll(
```

Comment:
A short TSDoc for these functions might be nice too.

Reply:
f4dca49a0966b8fa16418eee03810b890ba4fe42